app/javascript/components/student/editor/Submitting.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useCallback } from 'react'
+
+export function Submitting({ onCancel }: { onCancel: () => void }) {
+  const handleCancel = useCallback(
+    (e) => {
+      e.preventDefault()
+      onCancel()
+    },
+    [onCancel]
+  )
+
+  return (
+    <div>
+      <p>Submitting...</p>
+      <button onClick={handleCancel}>Cancel</button>
```

Comment:
If you add `type="button"`, you don't need to "preventDefault".

Reply:
This is cool!